examples/posix_sockets/Makefile

```diff
@@ -16,6 +16,7 @@ BOARD_INSUFFICIENT_MEMORY := airfy-beacon chronos msb-430 msb-430h nrf51dongle n
 # NOTE: 6LoWPAN will be included if IEEE802.15.4 devices are present
 USEMODULE += gnrc_netdev_default
 USEMODULE += auto_init_gnrc_netif
+USEMODULE += gnrc_txtsnd
```

Comment:
Well because typing `help` at the moment includes that command appears (but that should be the case for any application that includes `gnrc_netif`), so yes: please update README.